digitransit-component/packages/digitransit-component-suggestion-item/src/index.js

```diff
@@ -517,11 +523,32 @@ SuggestionItem.propTypes = {
     address: PropTypes.string,
     selectedIconId: PropTypes.string,
     iconColor: PropTypes.string,
+    translatedText: PropTypes.string,
+    properties: PropTypes.shape({
+      layer: PropTypes.string,
+      color: PropTypes.string,
+      localadmin: PropTypes.string,
+      mode: PropTypes.string,
+      id: PropTypes.string,
+      source: PropTypes.string,
+      arrowClicked: PropTypes.bool,
+      destination: PropTypes.shape({
+        name: PropTypes.string,
+        localadmin: PropTypes.string,
+      }),
+      origin: PropTypes.shape({
+        name: PropTypes.string,
+        localadmin: PropTypes.string,
+      }),
+    }),
```

Comment:
I wonder if these props are always populated? SuggestionItem might be a route like 'M1'. browser's debugger does not complain but I am wondering if props should be a minimal common set of fields, instead of all possible values which might be present.